lib/client.js

```diff
@@ -14,10 +14,10 @@ function rawPostMessage(client, msg, forceReady) {
   }
 }
 
-function wrappedPostMessage(name) {
+// params is an array for get/invoke or an object for set
+function wrappedPostMessage(name, params) {
```

Comment:
I realise that this is an existing issue, but I don't like the fact the `params` array represents multiple calls for `get` but a single call for `invoke`.